src/css/components/_progress.scss

```diff
@@ -40,6 +44,10 @@
   font-size: 1.666666666666666666em;
 }
 
+.video-js .vjs-progress-control:hover .vjs-progress-holder.disabled {
```

Comment:
I don't think this block is necessary and it causes the time display not to look right
![screen shot 2017-10-26 at 11 25 00](https://user-images.githubusercontent.com/535884/32061860-d5c06714-ba40-11e7-84da-b174d0faafe3.png)

Though, now that I think about it, this is probably on purpose so that the progress holder/seek handle isn't as large?